src/axolotl/cli/__init__.py

```diff
@@ -112,37 +121,52 @@ def do_inference(
 
     while True:
         print("=" * 80)
-        # support for multiline inputs
-        instruction = get_multi_line_input()
-        if not instruction:
+        if prompts:
+            if nPrompt < len(prompts):
+                instruction = prompts[nPrompt]
+                nPrompt += 1
+            else:
+                return
+        else:
+            instruction = get_multi_line_input()
+        # Run inference only on one device
+        if not instruction or cfg.device != "cuda:0":
```

Comment:
what is the justification running inference only on a single GPU?

Reply:
I get inference performed multiple times while running it with 4 GPUS: accelerate launch --multi_gpu --num_processes=4 --num_machines=1 \
--mixed_precision='no' --dynamo_backend='no' \
--module axolotl.cli.inference lora13b-chat.yml \